### PR TITLE
Add swift

### PR DESCRIPTION
**Pull request by theklausster**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
-## 0.1.2
```
Comment (JFreakDK) [inline CHANGELOG.md]: Revert this file

File: LICENSE [hunk 2]
```diff
@@ -1,5 +1 @@
-Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
```
Comment (JFreakDK) [inline LICENSE]: Revert this file

File: README.md [hunk 3]
```diff
@@ -1,113 +1,14 @@
-What is geofencing?
```
Comment (JFreakDK) [inline README.md]: Revert this file

File: example/ios/Runner.xcodeproj/xcshareddata/xcschemes/Runner.xcscheme [hunk 4]
```diff
@@ -65,7 +65,7 @@
       </AdditionalOptions>
    </LaunchAction>
    <ProfileAction
-      buildConfiguration = "Release"
```
Comment (JFreakDK) [inline example/ios/Runner.xcodeproj/xcshareddata/xcschemes/Runner.xcscheme]: Should this change?

File: pubspec.yaml [hunk 5]
```diff
@@ -1,17 +1,60 @@
 name: flutter_geofencing
-description: A geofencing plugin for Flutter applications - Works on both Android and iOS - supports Android X - based on Ben Konyi's Geofencing.
```
Comment (JFreakDK) [inline pubspec.yaml]: Revert these lines

File: pubspec.yaml [hunk 6]
```diff
@@ -1,17 +1,60 @@
...
+# following page: https://dart.dev/tools/pub/pubspec
+
+# The following section is specific to Flutter.
 flutter:
+  # This section identifies this Flutter project as a plugin project.
+  # The androidPackage and pluginClass identifiers should not ordinarily
+  # be modified. They are used by the tooling to maintain consistency when
+  # adding or updating assets for this project.
   plugin:
-    androidPackage: io.flutter.plugins.geofencing
```
Comment (JFreakDK) [inline pubspec.yaml]: Add this back

File: CHANGELOG.md (hunk 1 above)
Comment (theklausster) [reply CHANGELOG.md]: done

File: example/ios/Runner.xcodeproj/xcshareddata/xcschemes/Runner.xcscheme (hunk 4 above)
Comment (theklausster) [reply example/ios/Runner.xcodeproj/xcshareddata/xcschemes/Runner.xcscheme]: This is generated stuff - I'll try to revert it

Comment (theklausster) [reply example/ios/Runner.xcodeproj/xcshareddata/xcschemes/Runner.xcscheme]: I think this is Okay

File: pubspec.yaml (hunk 5 above)
Comment (theklausster) [reply pubspec.yaml]: done

File: pubspec.yaml (hunk 6 above)
Comment (theklausster) [reply pubspec.yaml]: done

File: README.md (hunk 3 above)
Comment (theklausster) [reply README.md]: done

File: LICENSE (hunk 2 above)
Comment (theklausster) [reply LICENSE]: done

**Pull request merged**